extensions/wikia/ExactTargetUpdates/tasks/ExactTargetUserTaskHelper.php

```diff
@@ -349,4 +362,16 @@ public function mergeUsersEditsData( array &$aUsersEditsData, array $aUserEditsD
 		}
 	}
 
+	/**
+	 * Extracts list of users ids from provided $aUsersData
+	 * @param $aUsersData array of users data arrays consistent with structure of ExactTargetUserHooksHelper::prepareUserParams
+	 * @return array
+	 */
+	public function prepareUserIds( $aUsersData ) {
+		foreach( $aUsersData as $aUserData ) {
+			$aUserIds[] = $aUserData['user_id'];
+		}
```

Comment:
If you want to use a new builtin method we got with the upgrade from PHP 5.4 to 5.6, [`array_column`](http://php.net/manual/en/function.array-column.php) will do this in one line: `array_column( $aUserData, 'user_id' );`